news/2024/2024-09-07-new-featured-artist-cast-heal.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: cast heal"
+date: 2024-09-07 20:00:00 +0000
+---
+
+Is **cast heal** our newest Featured Artist? We'll let you figure it out.
+
+![](https://assets.ppy.sh/artists/436/header.jpg)
+
+When we asked **cast heal** for some of their achievements and milestones for this article, they responded with "filthy runoff", so that should give you a solid idea of what you're getting into.
+
+**x** melancholic drum and bass tracks are officially part of [**cast heal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/436), all of which are pre-timed so your impulsive mapping decisions won't be blocked by anything.
```

Comment:
```suggestion
**10** melancholic drum and bass tracks are officially part of [**cast heal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/436), all of which are pre-timed so your impulsive mapping decisions won't be blocked by anything.
```

Reply:
one track was added at the last minute oops